lua/conform/util.lua

```diff
@@ -193,6 +193,13 @@ M.parse_rust_edition = function(dir)
   local manifest = vim.fs.find("Cargo.toml", { upward = true, path = dir })[1]
   if manifest then
     for line in io.lines(manifest) do
+      -- if a project is part of a workspace edition might be defined top-level
+      local matches = line:match("edition *= *{ *workspace *= *true *}")
+        or line:match("edition.workspace *= *true")
```

Comment:
A question that comes from not being familiar with cargo workspaces: do we really care if the value of `edition.workspace` is `true` or should we _always_ look for the value of the edition in parent `Cargo.toml` files if we fail to parse it from this file?

Reply:
I think we do care, i believe values are not implicitly read from upper toml, but not 100% sure atm.


In the coming week I’ll search for the right parts of the docs or try it out myself and report back.

Reply:
NOTE: I noticed that conform's default is 2021 which seems different than what rust chose.